ci-operator/step-registry/interop/tooling/deploy-selenium/README.md

```diff
@@ -0,0 +1,214 @@
+# interop-tooling-deploy-selenium-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Purpose](#purpose)
+- [Process](#process)
+  - [Prepare](#prepare)
+    - [Retrieve the console URL](#retrieve-the-console-url)
+    - [Check if SELENIUM\_NAMESPACE is defined](#check-if-selenium_namespace-is-defined)
+  - [Deploy](#deploy)
+    - [Create the Namespace](#create-the-namespace)
+    - [Deploy the Selenium Pod](#deploy-the-selenium-pod)
+    - [Create a Service](#create-a-service)
+    - [Create an Ingress](#create-an-ingress)
+    - [Create a Network Policy](#create-a-network-policy)
+  - [Verify](#verify)
+- [Container Used](#container-used)
+- [Requirements](#requirements)
+  - [Variables](#variables)
+  - [Infrastructure](#infrastructure)
+
+## Purpose
+
+To deploy a Selenium pod, along with the network infrastructure to use it from a different cluster, to the target test cluster. The container uses the [quay.io/redhatqe/selenium-standalone](https://quay.io/repository/redhatqe/selenium-standalone) image and allows us to use the container as a remote executor for Selenium tests.
+
+## Process
+
+This script can be separated into three parts: prepare, deploy, and verify.
+
+### Prepare
+
+#### Retrieve the console URL
```

Comment:
Can you please explain why the README files contain the code from the shell scripts?
I would suggest just adding a link (to avoid the need to change in multiple places)

Reply:
See [90600e36f457961ad427db2012821f66007e76db](https://github.com/openshift/release/pull/34094/commits/90600e36f457961ad427db2012821f66007e76db)